packages/rum-core/test/performance-monitoring/transaction-service.spec.js

```diff
@@ -354,4 +354,16 @@ describe('TransactionService', function() {
     const tr = customTrService.startTransaction('test', 'page-load')
     tr.detectFinish()
   })
+
+  it('should not capture breakdown metrics by default', done => {
+    config.events.observe(TRANSACTION_END, function() {
+      expect(tr1.breakdownTimings.length).toBe(0)
+      done()
+    })
+
+    const tr1 = transactionService.startTransaction('test1', 'custom')
+    const span1 = tr1.startSpan('span1', 'app')
+    span1.end()
```

Comment:
This might generate a span with duration 0, which would make the span to be ignored by the breakdown! 

The same with transaction which could have the duration as 0

Reply:
Not really, it would never create with duration 0 since we use performance.now under the hood and its in milliseconds. so it would always be > 0.